trl/utils.py

```diff
@@ -23,6 +24,8 @@ class ScriptArguments:
     Args:
         dataset_name (`str`):
             Dataset name.
+        dataset_config_name (`str` or `None`, *optional*, defaults to `None`):
```

Comment:
If you want to save a few keystrokes, I think this would also be fine
```suggestion
        dataset_config (`str` or `None`, *optional*, defaults to `None`):
```

Reply:
Agree, done in 34471d2